input/omnia_config.yml

```diff
@@ -104,3 +104,17 @@ csi_powerscale_driver_secret_file_path: ""
 # User need to download values.yaml file and fill required data in values.yaml file. Provided the path of the values.yaml file here.
 # mention configurable values
 csi_powerscale_driver_values_file_path: ""
+
+# Kubernetes Offline Install
+# If set to true, all packages/images will be pulled from OIM local repo.
+# If set to false, all packages/images will be pulled from source via internet.
+# It can either be true or false
+# Default value is true, means packages/images will be pulled from OIM local repo
+k8s_offline_install: true
```

Comment:
missing validation of this variable in cluster_validation role

Reply:
corrected